src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.csproj

```diff
@@ -13,8 +13,9 @@
     <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
 
     <!--
-      This project compiles against Microsoft.AspNetCore.App from the SDK.
-      This ensures that it's packaging output is correct and does not include local artifacts.
+      Act as if this project will compile against the Microsoft.AspNetCore.App from the SDK. This ensures that its
+      packaging output is correct and does not include local artifacts. But, see last-minute fixup in the
+      _UpdateSharedFrameworkAssemblies.target.
     -->
     <UseAspNetCoreSharedRuntime>true</UseAspNetCoreSharedRuntime>
     <DoNotApplyWorkaroundsToMicrosoftAspNetCoreApp>true</DoNotApplyWorkaroundsToMicrosoftAspNetCoreApp>
```

Comment:
@pranavkm this project is a mess and I don't remember why it needs these hacks. the .nuspec will contain just the framework reference no matter what we do (due to the `RemoveSharedFrameworkOnlyRefsFromNuspec` fixes in eng/AfterSolutionBuild.targets) and anything that lands in the .deps.json file (or whatever) will be overridden using the targeting pack's PackageOverrides.txt file in user's projects i.e. packages won't be downloaded.

Please remind me: Why do we need this unique special case❔